scenario_player/utils/token.py

```diff
@@ -169,5 +169,4 @@ def save_token_configuration_to_file(token_file: str, token_data: TokenDetails)
 
 
     """
-    with open(token_file) as handler:
-        json.dump(token_data, handler)
+    token_file.write_text(json.dumps(token_data))
```

Comment:
Nice, didn't know that existed.

Reply:
Pathlib is one of my favorite improvements from Python 3.